ci/jenkins/test.sh

```diff
@@ -677,6 +892,10 @@ if [[ ${TESTCASE} =~ "windows" ]]; then
 elif [[ ${TESTCASE} =~ "e2e" ]]; then
     deliver_antrea
     run_e2e
+elif [[ ${TESTCASE} =~ "multicluster" ]]; then
```

Comment:
you need to add a new line [here](https://github.com/antrea-io/antrea/blob/main/CONTRIBUTING.md#getting-your-pr-verified-by-ci) like below:
```
/test-multicluster-e2e: Multi-cluster e2e tests
```

Reply:
We should use the same name prefix for e2e and integration tests, not one with "test-multicluster-", another with "test-mc-"